_posts/2019-11-01-snowplow-r117-biskupin/index.md

```diff
@@ -0,0 +1,137 @@
+---
+layout: post
+title-short: "Snowplow R117 release"
+title: "Snowplow R117 release"
+tags: [snowplow, release]
+author: Łukasz Indykiewicz
+category: Releases
+permalink: /blog/2019/11/01/snowplow-r117-biskupin/
+---
+
+Snowplow release R117 is named after [Biskupin](https://en.wikipedia.org/wiki/Biskupin), which is "an archaeological site and a life-size model of a late Bronze Age fortified settlement in north-central Poland that also serves as a archaeological open-air museum".
+
+This release focuses on delivering small, but necessary improvements to Snowplow ecosystem:
+
+1. [TLS port binding and certificate](#tls-port-binding-and-certificate)
+2. [Referer parser refreshment](#referer-parser-refreshment)
+3. [IPv6 anonymization](#ipv6-anonymization)
+4. [Additional event fingerprint hashing methods (SHA1, SHA256, SHA384, SHA512)](#additional-event-fingerprint-hashing-methods)
+5. [Support to the spot market for core instances](#support-to-the-spot-market-for-core-instances)
+
+The `bad row` release has been renamed to R118 - check [roadmap](#roadmap) for more details.
+
+## TLS port binding and certificate
+
+### TLS/SSL Certificate termination
+As an additional security measure it is now possible to terminate TLS/SSL connection directly within `scala-stream-collector` using a battle-tested [lightben/ssl-config](https://lightbend.github.io/ssl-config/index.html).
+We introduce several new configuration parameters in order to fit most workflows and configurations.
```

Comment:
Maybe update with "We have introduced several new configuration parameters in order to accommodate different workflows and configurations"